rule/gen_testdata_test.go

```diff
@@ -147,8 +147,7 @@ func auditctlExec(t testing.TB, command string) (string, []byte) {
 	// Replace paths with ones in a temp dir for test environment consistency.
 	command = makePaths(t, tempDir, command)
 
-	args := strings.Fields(command)
-	_, err = exec.Command("auditctl", args...).Output()
+	_, err = exec.Command("sh", "-c", "auditctl "+command).Output()
```

Comment:
I'm wondering why this shells out. Could `makePaths` return a `[]string` of args rather than doing the final join?

Reply:
Originally I had made it shell out before implementing the parsing, then I just left it because I figured it help detect issues with the our own command line parsing, but that doesn't seem like it's going to be a problem. I like your suggestion so I'll make that change.